components/kyma-environment-broker/hack/scripts/vars.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
```

Comment:
Really want to commit this file?

In general, you could move your provisioning scripts to the installation directory of the whole compass.